doc/releases/release-notes-3.2.rst

```diff
@@ -502,10 +502,92 @@ Drivers and Sensors
 Networking
 **********
 
-* ``CONFIG_NET_CONFIG_IEEE802154_DEV_NAME`` has been removed in favor of
-  using a Devicetree choice given by ``zephyr,ieee802154``.
-
-* Added new Wi-Fi offload APIs for retrieving status and statistics.
+* CoAP:
+
+  * Replaced fixed CoAP retransmission count and acknowledgment random factor
```

Comment:
Should this either Replaced or fixed. Replaced fixed is unclear if this was replaced or fixed

Reply:
By "fixed" I've meant "predetermined", not "repaired", I see how this may sound a bit unfortunate though. I'll rephrase this part.